docs/contributing.rst

```diff
@@ -92,12 +92,11 @@ the following::
     $ mkdir -p ~/pyenv/zarr-dev
     $ python -m venv ~/pyenv/zarr-dev
     $ source ~/pyenv/zarr-dev/bin/activate
-    $ pip install -r requirements_dev_minimal.txt -r requirements_dev_numpy.txt
-    $ pip install -e .[docs]
+    $ pip install -e .[test,docs]
 
 To verify that your development environment is working, you can run the unit tests::
 
-    $ python -m pytest -v zarr
+    $ python -m pytest -v tests
```

Comment:
It looks like there's been a `src`/`tests` split since this was written, so the original recommendation no longer applies. This should work and be portable, but feel free to replace with whichever incantation is preferred!

Reply:
I think the standard invocation is `hatch env run run` (perhaps we should change this to `hatch env run tests`?)